srcpkgs/yacy/files/yacy/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec chpst -u yacy:yacy /opt/yacy/startYACY.sh -f
```

Comment:
Are there other options users could pass to the start script? You can include a check for a conf file and source it if available. Several other services have this option.

Reply:
Maybe the -s switch here below has some potential. could set to ~/.yacy or something.

`/opt/yacy/startYACY.sh --help
startscript for YaCy on UNIX-like systems
Options
  -h, --help		show this help
  -t, --tail-log	show the output of "tail -f DATA/LOG/yacy00.log" after starting YaCy
  -l, --logging		save the output of YaCy to yacy.log
  -d, --debug		show the output of YaCy on the console and enable remote monitoring with JMX
  -f, --foreground	run as a foreground process, showing the output of YaCy on the console
  -p, --print-out	only print the command, which would be executed to start YaCy
  -s, --startup [data-path] start YaCy using the specified data folder path, relative to the current user home
  -g, --gui		start a gui for YaCy
[jason@box ~]$ ls /opt/yacy
addon    build.properties  DATA                 gpl.txt                        killYACY.sh  libbuild  reconfigureYACY.sh  startYACY.bat        stopYACY.bat                     updateYACY.sh
AUTHORS  build.xml         defaults             htroot                         langdetect   locales   skins               startYACY_debug.bat  stopYACY.sh                      yacy.yellow
bin      COPYRIGHT         getWin32MaxHeap.bat  installYaCyWindowsService.bat  lib          NOTICE    source              startYACY.sh         uninstallYaCyWindowsService.bat
[jason@box ~]$ ls /opt/yacy/DATA/
ARCHIVE  CLASSIFICATION  DICTIONARIES  HTCACHE  HTDOCS  INDEX  LISTS  LOG  RELEASE  SETTINGS  SKINS  SURROGATES  WORK  yacy.running`

Reply:
`/opt/yacy/startYACY.sh --help
startscript for YaCy on UNIX-like systems
Options
  -h, --help		show this help
  -t, --tail-log	show the output of "tail -f DATA/LOG/yacy00.log" after starting YaCy
  -l, --logging		save the output of YaCy to yacy.log
  -d, --debug		show the output of YaCy on the console and enable remote monitoring with JMX
  -f, --foreground	run as a foreground process, showing the output of YaCy on the console
  -p, --print-out	only print the command, which would be executed to start YaCy
  -s, --startup [data-path] start YaCy using the specified data folder path, relative to the current user home
  -g, --gui		start a gui for YaCy

[jason@box ~]$ ls /opt/yacy
addon    build.properties  DATA                 gpl.txt                        killYACY.sh  libbuild  reconfigureYACY.sh  startYACY.bat        stopYACY.bat                     updateYACY.sh
AUTHORS  build.xml         defaults             htroot                         langdetect   locales   skins               startYACY_debug.bat  stopYACY.sh                      yacy.yellow
bin      COPYRIGHT         getWin32MaxHeap.bat  installYaCyWindowsService.bat  lib          NOTICE    source              startYACY.sh         uninstallYaCyWindowsService.bat

[jason@box ~]$ ls /opt/yacy/DATA/
ARCHIVE  CLASSIFICATION  DICTIONARIES  HTCACHE  HTDOCS  INDEX  LISTS  LOG  RELEASE  SETTINGS  SKINS  SURROGATES  WORK  yacy.running`